plugin/remotecall/main.go

```diff
@@ -4,15 +4,18 @@ import (
 	"fmt"
 
 	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
 	"github.com/tg123/sshpiper/libplugin"
 	"github.com/tg123/sshpiper/plugin/internal/remotecall"
-	"github.com/urfave/cli/v2"
 )
 
 func createRemoteCaller(c *cli.Context) (*remotecall.RemoteCall, error) {
 	remoteCall, err := remotecall.InitRemoteCall(
 		c.String(userClusterMappingEndpoint),
 		c.String(userClusterMappingEndpointToken),
+		c.Bool(userClusterMappingEndpointIsSocket),
```

Comment:
Can we drop the boolean? if the `SocketEndpoint` is provided we can just assume it should be used

Reply:
i want to keep explicit issocket